infra/base-images/base-builder/compile_honggfuzz

```diff
@@ -20,6 +20,12 @@ echo "Skipping compilation; using precompiled honggfuzz"
 cp $SRC/honggfuzz/honggfuzz.a $LIB_FUZZING_ENGINE
 cp $SRC/honggfuzz/honggfuzz $OUT/
 
+# Set flags necessary for netdriver compilation
+export LIB_HF_NETDRIVER="-Wl,-u,LIBHFNETDRIVER_module_netdriver -Wl,--start-group $SRC/honggfuzz/libhfnetdriver/libhfnetdriver.a $SRC/honggfuzz/libhfcommon/libhfcommon.a -Wl,--end-group"
+
+export HF_ND_CXX='-DHFND_FUZZING_ENTRY_FUNCTION_CXX(x,y)=extern const char* LIBHFNETDRIVER_module_netdriver;const char** LIBHFNETDRIVER_tmp1 = &LIBHFNETDRIVER_module_netdriver;extern "C" int HonggfuzzNetDriver_main(x,y);int HonggfuzzNetDriver_main(x,y)'
```

Comment:
s/ND/NETDRIVER/ for consistency

s/CXX/CXXFLAGS/
s/CC/CFLAGS/

as well so that we refer to these as e.g. HF_NETDRIVER_CFLAGS

Reply:
I opted to use `HFND_CFLAGS`, `HFND_CXXFLAGS` and `LIB_HFND` instead of `s/ND/NETDRIVER/` as `s/ND/NETDRIVER/` would also change the macro, and the macro is needed as is to be consistent with `hfuzz-clang` (https://github.com/google/honggfuzz/blob/60f01c26d9200a4a1058c35a8fc22441b63fa88c/hfuzz_cc/hfuzz-cc.c#L364-L373), which is needed if we want the fuzzers to be compatible  with honggfuzz netdriver fuzzers from outside OSS-Fuzz.